docs/sources/open-source.md

```diff
@@ -8,42 +8,64 @@ title: Open Source
 weight: 300
 ---
 
-# Open Source
+# Grafana OnCall Open Source guide
 
-We prepared three environments for OSS users:
-- **Hobby** environment for local usage & playing around: [README.md](https://github.com/grafana/oncall#getting-started).
+Grafana OnCall is a developer-friendly incident response tool that's available to Grafana open source and Grafana Cloud users. The OSS version of Grafana OnCall provides the same reliable on-call management solution along with the flexibility of a self-managed environment. 
+
+This guide describes the necessary installation and configuration steps needed to configure OSS Grafana OnCall. 
+
+The intended audience for this guide includes:
+- Grafana open source admins who are responsible for deploying and configuring Grafana OnCall.
+- Grafana open source users who need to configure SMS and phone notifications using Grafana Cloud.
+
+## Install Grafana OnCall OSS
+
+There are three Grafana OnCall OSS environments available:
+
+- **Hobby** playground environment for local usage: [README.md](https://github.com/grafana/oncall#getting-started)
 - **Development** environment for contributors: [DEVELOPER.md](https://github.com/grafana/oncall/blob/dev/DEVELOPER.md)
-- **Production** environment for reliable cloud installation using Helm: [Production Environment](#production-environment)
+- **Production** environment for reliable cloud installation: [Production Environment](#production-environment)
```

Comment:
```suggestion
- **Production** environment for reliable Cloud installation: [Production Environment](#production-environment)
```
I don't know what "reliable" means. I think I know, but this doesn't really explain the difference between this all important environment and the others...also, the Cloud designation is confusing because we're talking about OSS.